### ISSUE TITLE
Previous Settings not cleared up?

### ISSUE THREAD

**Issue by verybadsoldier** [biscuits99/rp-video-manager#12]
I am just trying to understand how the video_manager works and there is a thing I don't understand. But it might be a bug:
When I install one preset, let's say `consoles-1080p`, then the file `/opt/retropie/configs/all/retroarch-core-options.cfg` will be created/copied.
When I then install another preset which does not copy a core-option file (for example `thorig-1080p`), then the  existing core-option is not deleted (but still in use).
So I guess that presets, that are not using core-options (or other config files), may still use mistakenly the config files from a previously installed preset?

I hope I could explain what I mean ;) Thanks!

**Comment by biscuits99** [comment]
Yes, for the retroarch-core-options it only really updates one setting related to nestopia (due to a colour issue I think) , but its probably best off left alone given that users may heavily tweak their file. I am probably best to remove reference to that file.
As you say, not all custom settings have this file set to update either.
